test/theme_test.rb

```diff
@@ -52,7 +52,7 @@ def test_templates
 
   def test_snippets
     @themes.each do |theme|
-      assert_equal(["snippets/product"], theme.snippets.map(&:name))
+      assert_equal(["snippets/product"], theme.snippets.map(&:name), theme.storage.class.name)
```

Comment:
The last arg of `assert_equal` is the error message. Is this what you want? Might be leftover debug code?

Reply:
Yeah. Left over debug code to tell me which storage implementation was causing issues.